test/utils/generateAddress/index.ts

```diff
@@ -1,78 +1,73 @@
 import assert from 'assert-diff'
 import responses from '../../fixtures/responses'
-import {TestSuite} from '../../utils'
-import {GenerateAddressOptions} from '../../../src/offline/generate-address'
+import {generateXAddress, GenerateAddressOptions} from '../../../src/utils/generateAddress'
+import xAddressResponse from '../../fixtures/responses/generate-x-address.json'
```

Comment:
I suggest you just use `responses.generateAddress` instead of directly importing the JSON - you're using `responses` anyways, so it helps keep everything symmetrical